src/Slider/slider-style.scss

```diff
@@ -6,13 +6,10 @@ $disabled-selected: $gray-500;
 $disabled-unselected: #cccccc;
 $disabled-thumb-bg: #9a9a9a;
 $disabled-thumb-border-color: #8c8c8c;
-
-@mixin transition($props) {
-    -webkit-transition: $props;
-    -moz-transition: $props;
-    -o-transition: $props;
-    transition: $props;
-}
+$slider-width-normal: 24px;
+$slider-height-normal: $slider-width-normal;
+$slider-width-alternative: 56px;
+$slider-height-alternative: 32px;
```

Comment:
2rem

Reply:
> then I should change all the sized in this file to rem. I can't change just this one


so what's the verdict? Change all or keep all? @kherP @DanSallau @yousifalraheem ?

Reply:
`rem` is meant to be used to have a dynamic size when using web readers. I don't think we need to increase the dimensions of a slider when the user increase the font size. It will blow it out of proportions.